app/assets/javascripts/app/lib/app_post/z_searchable_multiple_select.coffee

```diff
@@ -0,0 +1,42 @@
+class App.SearchableMultipleSelect extends App.SearchableAjaxSelect
+
+  render: ->
+    @attribute.valueName = ''
+
+    @html App.view('generic/searchable_select_multiply')
```

Comment:
Please use the same name for the template as the class name (`searchable_select_multiply` => `searchable_multiple_select`).